contributions/lindarodgers.md

```diff
@@ -0,0 +1,31 @@
+# Linda's contribution
+[Linda Rodgers] https://github.com/lindarodgers
+![My fav octocdat](https://octodex.github.com/images/yogitocat.png)
+
+## My favorite quote
+> 'Today is a new day.  Let's rejoice and be glad in it.'
+
+## Two Truths and a lie
+
+- i play basketball
```

Comment:
This is the lie!

Here's how to strikethought `~~` on each side.

- ~~i play basketball~~

Reply:
Bold: cmd+B in VS Code or `*` on each side

*this is bold*

Reply:
Italic: `_` on each side

_this is italic_